biocrnpyler/plotting.py

```diff
@@ -322,7 +323,26 @@ def generate_networkx_graph(CRN, useweights=False, use_pretty_print=False, pp_sh
             CRNgraph.nodes[allnodenum]["k"] = str(rxn.propensity_type.k)
             CRNgraph.nodes[allnodenum]["k_r"] = ''
 
-        default_color = "blue"
+        reaction_color = None
+        if repr(rxn) in colordict:
+            reaction_color = colordict[repr(rxn)]
+        for k,p in rxn.propensity_type.propensity_dict["parameters"].items():
```

Comment:
Can you add a comment somewhere, maybe in the main plotting function docstring, that explains how colors are searched for?